rabbithole_test.go

```diff
@@ -265,6 +265,20 @@ var _ = Describe("Rabbithole", func() {
 		})
 	})
 
+	Context("PUT /cluster-name", func() {
+		It("Set cluster name", func() {
```

Comment:
Ok, no problem. BTW, since cluster name is defined at run time and is hostname based, it may be more elegant and predictable to set arbitrarily a value in ./bin/ci/before_build.sh. Are you ok with that ?